app/command.go

```diff
@@ -158,7 +158,7 @@ func (a *App) ExecuteCommand(args *model.CommandArgs) (*model.CommandResponse, *
 
 	clientTriggerId, triggerId, appErr := model.GenerateTriggerId(args.UserId, a.AsymmetricSigningKey())
 	if appErr != nil {
-		mlog.Error(appErr.Error())
+		mlog.Error("error occurred in generating trigger Id for a user ", mlog.Err(appErr))
```

Comment:
@reflog Do we have guideline in creating message?  Like, should it start in capital letter "Error occurred ..."?

Reply:
@saturninoabril - not that I'm aware of. Aspecially for 'internal' errors that are not translated or bubbled up to the user.
@Lumexralph - please sync the branch with master and we'll be able to merge

Reply:
@reflog done!